addons/sale_loyalty/models/sale_order.py

```diff
@@ -176,6 +184,21 @@ def action_open_reward_wizard(self):
             return True
         return self.env['ir.actions.actions']._for_xml_id('sale_loyalty.sale_loyalty_reward_wizard_action')
 
+    def action_view_gift_cards(self):
+        self.ensure_one()
+        return {
+            'name': 'Gift Cards',
+            'type': 'ir.actions.act_window',
+            'view_mode': 'list',
+            'res_model': 'loyalty.card',
+            'domain': [
+                ('order_id', '=', self.id),
+                ('program_type', '=', 'gift_card')
+            ],
+            'context': "{'create': False}",
```

Comment:
```suggestion
            'context': '{'create': False}',
```

Reply:
Only this -> `'context': {'create': False},` works :+1: